modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -5170,58 +5340,120 @@ public ObjectDefinitionModel newObjectDefinitionModel(long objectFolderId) {
 		objectDefinitionModel.setCreateDate(new Date());
 		objectDefinitionModel.setModifiedDate(new Date());
 
-		// Other field
+		// Other fields
 
-		objectDefinitionModel.setDescriptionObjectFieldId(0);
 		objectDefinitionModel.setObjectFolderId(objectFolderId);
-		objectDefinitionModel.setRootObjectDefinitionId(0);
-		objectDefinitionModel.setTitleObjectFieldId(_counter.get());
-		objectDefinitionModel.setAccountEntryRestricted(false);
 		objectDefinitionModel.setActive(true);
-		objectDefinitionModel.setLabel(
-			StringBundler.concat(
-				"<?xml version=\"1.0\"?><root available-locales=\"en_US\" ",
-				"default-locale=\"en_US\"><Label language-id=\"en_US\">",
-				"Commerce Order</Label></root>"));
-		objectDefinitionModel.setClassName(CommerceOrder.class.getName());
+		objectDefinitionModel.setDBTableName(
+			StringBundler.concat("O_", _companyId, StringPool.UNDERLINE, name));
```

Comment:
Is there a way to reuse our existing logic here? Using methods like `ObjectDefinitionLocalServiceImpl#_getDBTableName`.
I'm afraid that it will break the tests if we change this logic in the object code, as the change will not be reflected here.

Reply:
Yes, in the future we may try to move this logic to the ObjectDefinition entity.